docs/topics/proc_running-the-secured-booster-integration-tests-on-sb-vx.adoc

```diff
@@ -1,10 +1,11 @@
 
-[#running-the-secured-booster-integration-tests-on-vertx]
-= Running the {name-mission-secured} Booster Integration Tests on {VertX}
+[#running-the-secured-booster-integration-tests-on-{parameter-runtime}]
```

Comment:
Sorry, I forgot, can you please add the following comment to the file?

```
// This is a parameterized module. Parameters used: 
// 
//   parameter-$SOMETHING: $DESCRIPTION_OF_THE_PARAMETER
//   parameter-$SOMETHING_ELSE: ...
//
// Rationale: $YOUR_RATIONALE
```

I will merge it then.